packages/server/src/ChunkExtractor.js

```diff
@@ -340,7 +365,10 @@ class ChunkExtractor {
     const requiredScriptTag = this.getRequiredChunksScriptTag(extraProps)
     const mainAssets = this.getMainAssets('script')
     const assetsScriptTags = mainAssets.map(asset =>
-      assetToScriptTag(asset, extraProps),
+      assetToScriptTag({
+        ...asset,
+        ...this.getSriFromFileName(asset.filename),
```

Comment:
All of these is useless, the integrity should already be in asset. The only needed part is : getSriHtmlAttributes and all usage of it.